client/state/sites/selectors.js

```diff
@@ -985,3 +985,19 @@ export function getCustomizerUrl( state, siteId ) {
 		'return': returnUrl
 	}, adminUrl );
 }
+
+/*
+ * Returns true if the site has unchanged site title
+ *
+ * @param {Object} state Global state tree
+ * @param {Object} siteId Site ID
+ * @return {Boolean} True if site title is default, false otherwise.
+ */
+export const hasDefaultSiteTitle = ( state, siteId ) => {
+	const site = getRawSite( state, siteId );
+	if ( ! site ) {
+		return null;
+	}
+	const slug = getSiteSlug( state, siteId );
+	return site.name === i18n.translate( 'Site Title' ) || slug.indexOf( site.name ) === 0;
```

Comment:
Why the `indexOf`? This means "if the site slug starts with site.name", doesn't it? What's the intent here?